bloom/generators/rpm/templates/ament_cmake/template.spec.em

```diff
@@ -56,6 +57,18 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
 if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi
 %make_install -C obj-%{_target_platform}
 
+%if 0%{?with_tests}
+%check
+TEST_TARGET=$(%__make -qp -C obj-%{_target_platform} | sed "s/^\(test\|check\):.*/\\1/;t f;d;:f;q0")
+if [ -n "$TEST_TARGET" ]; then
+# In case we're installing to a non-standard location, look for a setup.sh
+# in the install tree that was dropped by catkin, and source it.  It will
```

Comment:
`catkin` in an `ament_cmake` template seems like a copy-n-paste?

Same below for `ament_python`.

Reply:
I'm not going to break the pattern. Merge #588 if you want it changed.